books/free-programming-books-langs.md

```diff
@@ -1654,6 +1655,11 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Octave Programming](https://en.wikibooks.org/wiki/Octave_Programming_Tutorial) - Wikibooks
 
 
+### Odin
+
+* [The Odin language overview](https://odin-lang.org/docs/overview/)
```

Comment:
Don't invent titles! Use the title given by the resource. In this case `Overview | Odin Programming Language`. (check other titles to see how to properly handle the pipe character in markdown.)

Reply:
At https://odin-lang.org/docs/ it is described as 'An overview of the Odin programming language and its features.', would this be valid as a name?

Reply:
That's a description, not a title. I took the title from the html title element

Reply:
I changed it to what you recommended, thank you for taking the time to review my submission. :)

Reply:
markdown wants you to use `\|`

Reply:
I just noticed it as well, I already made the change